### PR TITLE
support image normalization

### PR DESCRIPTION
**Pull request by mengxue-rs**
In this commit, common normalization methods are realized in the normalise_image function of utils.py and a normalization argument is added in the main.py.

### CODE REVIEW

**Comment by mengxue-rs** [comment]
the default value of the normalization argument is set as MNI (Converting the dynamics to [0, 1] along the whole image.) same to original setting https://github.com/nshaud/DeepHyperX/blob/99792c5aec51a9602099b4b1c6618af01b652e09/datasets.py#L319

File: datasets.py [hunk 1]
```diff
@@ -96,7 +96,7 @@ def update_to(self, b=1, bsize=1, tsize=None):
         self.update(b * bsize - self.n)  # will also set self.n = b * bsize
 
 
-def get_dataset(dataset_name, target_folder="./", datasets=DATASETS_CONFIG):
+def get_dataset(dataset_name, normalization_method, target_folder="./", datasets=DATASETS_CONFIG):
```
Comment (nshaud) [inline datasets.py]: Could you make this a keyword argument with a default value?

File: main.py [hunk 2]
```diff
@@ -129,6 +129,19 @@
...
+    type=str,
+    default='MNI',
+    help="Normalization method to use for image preprocessing. Available:\n"
+    "None: Applying none preprocessing."
+    "L2NS: Normalizing with a unit Euclidean norm along each sample."
+    "MNB: Converting the dynamics to [0, 1] along each band."
+    "SNB: Normalizing first- and second-order moments along each band."
+    "MNI: Converting the dynamics to [0, 1] along the whole image."
+    "SNI: Normalizing first- and second-order moments along the whole image.",
+)
```
Comment (nshaud) [inline main.py:143]: Can you add what these acronyms mean?

**Review by nshaud — COMMENTED** [review]
Good job. I added a few comments to clarify some points but we will be able to merge this soon.

**Pull request closed without merging**